src/dragging.js

```diff
@@ -0,0 +1,22 @@
+let droppedItem;
+let held;
+function heldItem(e) {
+  held = parseInt(e.target.id, 10);
+}
+
+function droppedOn(e) {
+  e.preventDefault();
+  droppedItem = parseInt(e.target.id, 10);
+}
+
+function dragger(arr) {
+  const item1 = arr[held];
+  const item2 = arr[droppedItem];
+  arr.splice(item1.index, 1);
+  arr.splice(item2.index, 0, item1);
+  for (let i = 0; i < arr.length; i += 1) {
+    arr[i].index = i;
+  }
+}
```

Comment:
- [ ] It looks like you have the dragger function, but in element, I couldn't be able to see where to change the order of the elements, I kindly invite you to do it. 🐱 
![image](https://user-images.githubusercontent.com/20563252/125881295-070efe19-4a53-43a5-a60a-36becea75932.png)